numba/tests/test_random.py

```diff
@@ -1374,6 +1380,92 @@ def test_dirichlet_exceptions(self):
                 str(raises.exception),
             )
 
+class TestRandomNoncentralChiSquare(BaseTest):
+
+    def _check_sample(self, size, sample):
+
+        """Check output structure"""
+        if size is not None:
+            self.assertIsInstance(sample, np.ndarray)
+            self.assertEqual(sample.dtype, np.float64)
+            
+            if type(size) is int:
```

Comment:
In this case, I believe that `isinstance` is the preferred idiom over using `type`.